modules/channel-web/src/actions/sendPostbackToParent.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
I think we need should use two different terms for that. The postback is when a user clicks a button and a payload is 'posted back' to the server. 

Pushing something from the server, to the webchat, to talk to the parent window, is a completely different matter. Maybe something like 'browser-push' or 'web-push' or 'webclient-browser' or something similar?